docs/updating/updating_to_3.3.md

```diff
@@ -167,7 +168,49 @@ Check the Location ID of the "Components" Content item and set it as a value of
 If you are upgrading between [[= product_name_com =]] versions,
 add the `content/read` Policy with the Owner Limitation set to `self` to the "Ecommerce registered users" Role.
 
-### Updating to 3.3.2
+## Finish the update
+
+Finish the update by running the following commands:
+
+``` bash
+php bin/console ibexa:graphql:generate-schema
+composer run post-install-cmd
+```
+
+## Update to v3.3.x
+
+!!! note
+
+    You can only update to the latest patch release of 3.3.x.
+
+To update from one v3.3 patch version to another (for example, from v3.3.1 to v3.3.2), run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:3.3.2 --with-all-dependencies
+    composer recipes:install ibexa/content --force -v
```

Comment:
Since recipes might contain changes that are required for, for example, `cache:clear` command, we should call `composer require` with `--no-scripts` option to postpone them.

```suggestion
    composer require ibexa/content:3.3.2 --with-all-dependencies --no-scripts
    composer recipes:install ibexa/content --force -v
    composer run post-install-cmd
```

Discovered by @micszo

Reply:
It does not do the trick, we need to find a different solution 🙂